### PR TITLE
Update diversity, remove exclusions

### PR DESCRIPTION
**Pull request by ivdma**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ Classics are common patterns that return in Round The Table matches, noticably b
 - The player saving the other will loose its life if the player makes a mistake. This way there is always a risk involved helping others.
 
 ### Promoting Diversity
-Diversity at HackerOne is a hot topic right now. We aim to be as diverse as possible. Not only when working on world-class product, but also during the teambuilding. Therefore any diverse playing member (women, foreigners, visitors from SF) will start any round with one life. That excludes players who own Frisian passport, and players from Brabant.
+Diversity at HackerOne is a hot topic. We aim to be as diverse as possible. Not only when working on world-class product, but also during teambuilding. Therefore any diverse playing member (women, foreigners, visitors from SF) will start any round with one life. That excludes players who own Frisian passport, and players from Brabant.
```
Comment (Willianvdv) [inline README.md]: - last comma can be removed
- `on world-class` should be `on a world-class`

**Review by Willianvdv — DISMISSED** [review]

**Comment by Willianvdv** [comment]
@spidfire you like?

**Comment by spidfire** [comment]
Will Frisian and Brabandic players get no points or more?
Is it not better to give new players 1 point instead of these specific groups?

**Comment by ivdma** [comment]
@spidfire Frisian and Brabantic players are, sadly enough, mostly being referred to as 'foreigners', which is unfair. So to prevent the confusion we exclude those groups from being awarded with 1 life at the beginning of the round and only award really diverse players like me with it. That, hopefully, will attract more diverse people to @Hacker0x01 company.

**Comment by Willianvdv** [comment]
@spidfire excellent feedback, let's give all inexperienced players a beginners life. The big question, when do you become an experienced player?

**Comment by ivdma** [comment]
@Willianvdv @spidfire beating @mvgijssel for the first time in finals could draw that line.

**Comment by mvgijssel** [comment]
After playing around with this rule, is it an idea to lose the life before the final starts? In which case you also have to call "reset" if you make it? That way it's a little bit more fair towards the regular players.

**Comment by ivdma** [comment]
My suggestion: 
```ruby
# At the beginning of any round
lives ||= []
if lives.empty?
  lives << Life.new(type: :beginner)
# else
#   No new life 
end
```

**Comment by Willianvdv** [comment]
Can't we refactor this whole README into code? way easier

File: README.md [hunk 2]
```diff
@@ -93,7 +93,7 @@ Classics are common patterns that return in Round The Table matches, noticably b
 - The player saving the other will loose its life if the player makes a mistake. This way there is always a risk involved helping others.
 
 ### Promoting Diversity
-Diversity at HackerOne is a hot topic right now. We aim to be as diverse as possible. Not only when working on world-class product, but also during the teambuilding. Therefore any diverse playing member (women, foreigners, visitors from SF) will start any round with one life. That excludes players who own Frisian passport, and players from Brabant.
+Diversity at HackerOne is a hot topic. We aim to be as diverse as possible. Not only when working on a world-class product, but also during teambuilding. Therefore any diverse playing member (women, foreigners, visitors from SF) will start any round with one life. That excludes players who own Frisian passport and players from Brabant.
```
Comment (Willianvdv) [inline README.md]: remove the exclusions, everyone can have a beginners life 🥇

**Review by Willianvdv — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ivdma) [reply README.md]: So basically it's not about diversity anymore but about the level of the player. But then how do you decide what level the player has reached?

Comment (RSO) [reply README.md]: In comes reputation! I would suggest we add +7 points in the overall scoring for won matches, -5 for pulling a jeurissen, etc. @mvgijssel probably has some super strong opinions on what gem to use to implement reputation.

Comment (esjee) [reply README.md]: Sink or swim. You get it for 5 days, it's your responsibility to shape up during that time.

**Comment by spidfire** [comment]
Just wondering do you only get visitors from the SF office or also from one of your new offices?

**Comment by ivdma** [comment]
This information will remain undisclosed.

**Review by Willianvdv — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -93,7 +93,14 @@ Classics are common patterns that return in Round The Table matches, noticably b
 - The player saving the other will loose its life if the player makes a mistake. This way there is always a risk involved helping others.
 
 ### Promoting Diversity
-Diversity at HackerOne is a hot topic right now. We aim to be as diverse as possible. Not only when working on world-class product, but also during the teambuilding. Therefore any diverse playing member (women, foreigners, visitors from SF) will start any round with one life. That excludes players who own Frisian passport, and players from Brabant.
+Diversity at HackerOne is a hot topic. We aim to be as diverse as possible. Not only when working on a world-class product, but also during teambuilding. Therefore any diverse playing member (e.g. women, foreigners, visitors from other locations) will start any round with one extra beginners life. That applies only if starting first round or when player has one normal life. So basically it's:
+
+```ruby
+rounds.each do |round|
+  lives ||= [Live.new(type: :normal)]
+  lives << Live.new(type: :beginner) if lives.count == 1
```
Comment (Willianvdv) [inline README.md]:
```ruby
irb(main):001:0> 1.count
NoMethodError: undefined method `count' for 1:Fixnum
	from (irb):1
	from /Users/willian/.rbenv/versions/2.3.3/bin/irb:11:in `<main>'
```

**Pull request merged**